tests/phpunit/class-wc-stripe-checkout-sessions-ajax-handler-test.php

```
@@ -103,6 +103,92 @@ public function test_create_checkout_session(
+
+		$this->assertIsArray( $captured_request );
+		$this->assertArrayHasKey( 'line_items', $captured_request );
+		$this->assertCount( 1, $captured_request['line_items'] );
+
+		$expected_amount = WC_Stripe_Helper::get_stripe_amount( WC()->cart->get_total( 'edit' ), get_woocommerce_currency() );
+		$line_item       = $captured_request['line_items'][0];
+
+		$this->assertSame( __( 'Cart total', 'woocommerce-gateway-stripe' ), $line_item['price_data']['product_data']['name'] );
+		$this->assertSame( $expected_amount, $line_item['price_data']['unit_amount'] );
+		$this->assertSame( 1, $line_item['quantity'] );
+	}
```

PHPMD (2.15.0)

[error] test_create_checkout_session_sends_single_line_item_matching_cart_total accesses the super-global variable $_REQUEST. (undefined)

(Superglobals)